Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -0,0 +1,358 @@
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_SearchbyFileHash_Command(clientObj, sha1_checksum):
+    response = clientObj.get_searchbyFileHash(sha1_checksum)
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Search Results', responseJson['results'])
+        outputs = {"Druva.searchEndpointsFileHashResults(val.objectID == obj.objectID)": responseJson['results']}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_Restore_Endpoint(clientObj, source_resourceid, target_resourceid, restore_location):
+    response = clientObj.post_restoreToEndpoint(source_resourceid, target_resourceid, restore_location)
+    statusCode = response.status_code
+    responseJson = response.json()
+    if (statusCode == 200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Restore Job Initiated', responseJson['restores'])
+        outputs = {"Druva.restoreJobs(val.restoreID == obj.restoreID)": responseJson['restores']}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_Restore_Status(clientObj, restore_id):
+    response = clientObj.get_restoreStatus(restore_id)
+    statusCode = response.status_code
+    responseJson = response.json()
+    if (statusCode == 200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Restore Job Status', responseJson)
+        outputs = {"Druva.restoreJobs(val.restoreID == obj.restoreID)": responseJson}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_Decommission(clientObj, resource_id):
+    response = clientObj.post_decommission(resource_id)
+    statusCode = response.status_code
+    responseJson = response.json()
+    if (statusCode == 200):
+        responseJson = response.json()
+        headers = ['Resource ID']
+        readable_output = tableToMarkdown('Device Decomission Request', str(resource_id), headers=headers)
+        outputs = {"Druva.decomissionedResource(val.resource_id == obj.resource_id)": str(resource_id)}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def main():
+    command = demisto.command()
+    params = demisto.params()
```

Comment:
avoid calling `demisto.args()` over and over
```suggestion
    params = demisto.params()
    args = demisto.args()
```